modules/ossm-security-cert-manage.adoc

```diff
@@ -44,96 +45,68 @@ spec:
       istiod:
         type: PrivateKey
         privateKey:
-          rootCADir:  /etc/cacerts
+          rootCADir: /etc/cacerts
 ----
++
+[NOTE]
+====
+After creating/changing/deleting the `cacert` secret, the control plane `istiod` and `gateway` pods must be restarted so the changes go into effect. Use the following command to restart the pods: `oc -n istio-system delete pods -l 'app in (istiod,istio-ingressgateway)'`.
+
+Restart the application pods so that the sidecar containers pick up the changes as well.
+====
 
 [id="ossm-cert-manage-verify-cert_{context}"]
 == Verifying your certificates
 
-Use the Bookinfo sample application to verify your certificates are mounted correctly. First, retrieve the mounted certificates. Then, verify the certificates mounted on the pod.
+Use the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-tutorial-bookinfo-overview_ossm-create-mesh[Bookinfo example application] to verify that the workload certificates are signed by the certificates that were plugged into the CA. This requires you have `openssl` installed on your machine
 
-. Store the pod name in the variable `RATINGSPOD`.
-+
-[source,terminal]
-----
-$ RATINGSPOD=`oc get pods -l app=ratings -o jsonpath='{.items[0].metadata.name}'`
-----
+. Extracting certificates from bookinfo workloads
 +
-. Run the following commands to retrieve the certificates mounted on the proxy.
+Wait about 1 minute for the mTLS policty to take effect before retrieving the certificate chain of bookinfo. If Bookinfo was already deployed when you created the secret above, you need to restart the pods so they pick up the changes (see note above). Replace `<bookinfo-project>` with the project name where bookinfo workload is installed.
```

Comment:
The "sleep 60" line in the command takes care of this 1 minute wait.
If you move the pod restart to the previous topic, that removes the need for second sentence AND the note.

If we just use `bookinfo`  instead of "replace <bookinfo-project> it increases usability.  Let's just assume that customers are following the directions and didn't install this anywhere with a different name.